### DISCUSSION TITLE
Authentication of services and applications in AdvantEdge

### DISCUSSION THREAD

**Discussion by eduardolfalcao** [InterDigitalInc/AdvantEDGE#379] [Q&A] [answered]
Hi folks,

I was wondering whether AdvantEdge implements authentication between Edge Apps and MEC Services but managed to find nothing about it. Does the AdvantEdge implement this feature, or it simply abstracts security aspects such as authentication?

Best regards

**Comment by dilallkx** [comment, marked as answer]
Hi @eduardolfalcao,

Thank you for reaching out!

AdvantEDGE does not provide authentication between edge apps and MEC services. MEC Services are deployed as sandbox pods and are accessible to all edge applications deployed in the sandbox. See the [architecture documentation](https://interdigitalinc.github.io/AdvantEDGE/docs/overview/overview-architecture/#micro-service-architecture) for more information about the microservices running in a sandbox.

Regards!
